### PR TITLE
feat: add apollo compatibility mode in wgc federated-graph fetch command

### PR DESCRIPTION
**Pull request by JivusAyrus**
## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: cli/src/commands/graph/federated-graph/utils.ts [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+      "@tag"
+    ]
+  )
+
+`;
+  let hasDefinedLink = false;
+  if (!isV2Graph) {
+    return schema;
+  }
+  visit(parse(schema), {
```
Comment (Aenimus) [inline cli/src/commands/graph/federated-graph/utils.ts:205]: parse can fail, but I guess at this point we must have a valid schema

Comment (JivusAyrus) [reply cli/src/commands/graph/federated-graph/utils.ts:205]: yep

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: connect/src/wg/cosmo/platform/v1/platform_pb.ts [hunk 2]
```diff
@@ -2416,6 +2416,16 @@ export class Subgraph extends Message<Subgraph> {
...
 
+  /**
+   * @generated from field: string subscriptionProtocol = 11;
+   */
+  subscriptionProtocol = "";
+
+  /**
+   * @generated from field: optional bool isV2Graph = 12;
+   */
+  isV2Graph?: boolean;
```
[the remark this answers is not in the document]
Comment (Aenimus) [reply connect/src/wg/cosmo/platform/v1/platform_pb.ts:2427]: It's just federation v2 by anyone's standards including Cosmo/WunderGraph's.

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**